recipes/bashlex/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "bashlex" %}
+{% set version = "0.12" %}
+{% set sha256 = "94bcf8759f28286d4fe2bbc408fa50e9fb7c46fa7097eac5b7cd76d75ca9936d" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - enum34
+
+test:
+  imports:
+    - bashlex
+
+about:
+  home: https://github.com/idank/bashlex
+  license: GPL-3.0
+  license_family: GPL
```

Comment:
Please manually include the `license_file`:

https://conda-forge.org/docs/meta.html#packaging-the-license-manually

and open an issue upstream about including it in the source distribution. If you could also put a copy of the issue url in the `meta.yaml` as a comment that would be great. This way there is a reminder to check on the status of the issue next time the feedstock is updated.